docs/examples/user_types/nodejs/02-userClasses.js

```diff
@@ -29,23 +29,25 @@ oso.registerClass(InternalUser);
 class AccountManager extends InternalUser {
   customerAccounts() {
     return db.query(
-      'SELECT id FROM customer_accounts WHERE manager_id = ?',
+      "SELECT id FROM customer_accounts WHERE manager_id = ?",
       this.id
     );
   }
 }
 // account-end
 
 function userFromId(id) {
-  const userType = db.query('SELECT type FROM users WHERE id = ?', id);
-  if (userType === 'internal') {
+  const userType = db.query("SELECT type FROM users WHERE id = ?", id);
+  if (userType === "internal") {
     const actor = new InternalUser(id);
-    if (actor.role() === 'account_manager') {
+    if (actor.role() === "account_manager") {
       return new AccountManager(id);
     } else {
       return actor;
     }
-  } else if (userType === 'customer') {
+  } else if (userType === "customer") {
```

Comment:
Can you make it so your autoformatter doesn't change all of these quotes to double quotes? It would make it easier to see what the actual changes are in the JS files that were touched.